Kalela/Level-Up
# 155768857: Connect-recyclerview-to-mainactivity

# What does this PR do?
- Connects the RecycleView to the main activity.
- Sets up continuous integration with CircleCI.

### Screenshots
![image](https://user-images.githubusercontent.com/33759757/41976278-6593c422-7a25-11e8-8b16-488417c6c397.png)

### How should this be manually tested?
1. Clone the repo.
2. Open the project with android studio.
3. Run the app.

Check CircleCI test results in the readme badges.

### What are the relevant pivotal tracker stories?
#### 155768857
Connect RecyclerView layout File to the MainActivity
